docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -33,51 +33,103 @@ We suggest you have the following materials nearby:
 
 To ensure that your content shows up correctly on the link:https://quarkus.io/guides/[Quarkus documentation home page], use the following steps:
 
-. Decide on a content type that best fits the content that you are contributing.
+. Decide on a [diataxis](https://quarkus.io/guides/doc-concept) content type that best fits the content that you are contributing.
 +
 TIP: To help you decide, see the content type descriptions in xref:{doc-guides}/doc-reference.adoc#titles-and-headings[Titles and headings] on the "About Quarkus documentation" page.
-+
+
 . Go to the `src/main/asciidoc/_templates` directory, and make a copy of the relevant template for the content type you have chosen. Be sure to:
 ** Use the filename syntax of`<category>-<titlekeyword>-<titlekeyword>-<content-type>.adoc`. For example, `security-basic-authentication-tutorial.adoc`.
 ** Save the file to the `docs/src/main/asciidoc` folder in the `quarkus` repository.
-. Set the minimum required header information as outlined in the following example:
+
+. Set the minimum required header information to ensure that the content renders correctly in the website portal and on the documentation home page, as outlined in the following example:
 +
 [source,asciidoc]
 ----
 [id="security-basic-authentication-howto"] <1>
 = Secure a Quarkus application with basic authentication <2>
 include::_attributes.adoc[] <3>
 :categories: security,web <4>
+
```

Comment:
Why the extra line addition?

Reply:
Good question. @michelle-purcell pointed out that this blank line is necessary before the abstract.
I've added a callout that mentions it now.